### PR TITLE
[FIX] Transaction of Matic submitted in polygon network goes to the activity view

### PR DESCRIPTION
**Pull request by tommasini**
**Description**
The problem was that the submitted transactions in the Polygon Mainnet do not appear in the left option Activity View.
The network firstly was always loading, then the first submitted transaction was not appearing on the activity view.
The transactions were not appearing because we do not render the submitted transactions when we do not have any other listed transaction

**Proposed Solution**
The proposed solution it's showing as well the transactions submitted when we have no older transactions

**Code Impact**
Low, it will only impact the Activity View screen.

**Test Cases**
Case:1
- Go to settings -> Advanced
- Press reset account (to have no transaction on activity view)
- Go to polygon network
- transfer 0.001 matic
- go to activity view and verify if it's there
(Did the same for rinkeby network)
Case2:
- Tested transfer on rinkeby network
Case3:
- Tranfer of collectible on rinkeby network

**Checklist**

* [x] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

**Screenshots/Recordings**

https://user-images.githubusercontent.com/46944231/168602382-7cde981b-14c3-4d92-a8d4-416d84b18933.mp4

**Issue**

Progresses #3820

### CODE REVIEW

File: app/components/Views/TransactionsView/index.js [hunk 1]
```diff
@@ -42,88 +42,96 @@ const TransactionsView = ({
   const [submittedTxs, setSubmittedTxs] = useState([]);
   const [confirmedTxs, setConfirmedTxs] = useState([]);
   const [loading, setLoading] = useState();
+  const network = Engine.context.NetworkController.state.network;
```
Comment (gantunesr) [inline app/components/Views/TransactionsView/index.js:45]: Use hook `useSelector`

**Review by gantunesr — CHANGES_REQUESTED** [review]
Let's remove all the changes caused by tabs/spaces

**Review by gantunesr — APPROVED** [review]
LGTM

File: app/components/Views/TransactionsView/index.js (hunk 1 above)
Comment (tommasini) [reply app/components/Views/TransactionsView/index.js]: Implemented in this [commit](https://github.com/MetaMask/metamask-mobile/pull/4329/commits/23a62176a133ff274e8cc7f249625bea6c1ef394)

**Comment by chrisleewilcox** [comment]
LGTM!

**Review by chrisleewilcox — APPROVED** [review]
LGTM

**Pull request merged**